SPECS/conmon/conmon.spec

```diff
@@ -52,6 +51,9 @@ make PREFIX=%{buildroot}%{_prefix} install install.crio
 %dir %{_libexecdir}/crio
 
 %changelog
+* Fri Apr 05 2024 Sumedh Sharma <sumsharma@microsoft.com> - 2.1.2-1
+- Bump version to 2.1.2 needed for CVE-2022-1708
```

Comment:
The CVE seems to be impacting `cri-o` judging by [the advisory](https://github.com/advisories/GHSA-fcm2-6c3h-pg6j) and [one of the links](https://github.com/cri-o/cri-o/commit/f032cf649ecc7e0c46718bd9e7814bfb317cb544) from it, so I'm not sure patching `conmon` is sufficient.

Reply:
I had raised the PR for cri-o against main. My bad that I failed to raise it against fasttrack. https://github.com/microsoft/azurelinux/pull/8745 has been raised for the same.